.github/CODEOWNERS

```diff
@@ -0,0 +1,1859 @@
+####
```

Comment:
let's remove the CODEOWNERS as it has been intentionally removed from the 8.x branch